docs/examples/notebooks/Working_with_STAC_simple.ipynb

```diff
@@ -129,6 +129,7 @@
     "    f\"{titiler_endpoint}/stac/info\",\n",
     "    params = {\n",
     "        \"url\": stac_item,\n",
+    "        \"assets\": \"pan\"\n",
```

Comment:
I believe `assets` is a required parameter, looking at the `stac/info` docs

Reply:
Yes it is now, but the version deployed in api.cogeo.xyz is not up to date 😬